Console Spam on Bukkit/Spigot 1.15.2

### Type of bug

Error in console

### Error log (if applicable)

https://paste.gg/p/anonymous/244cf90baa4f45a48ab35c0f38206311

### Bug description

When I type the command or any mydog command, I get the console error.

### Steps to reproduce

Using the command

### Expected behaviour

Don't let that come out every time I write

### Actual behaviour

[20:13:46] [Server thread/ERROR]: Exception when VodkaTroika attempted to tab complete mydog list
org.bukkit.command.CommandException: Unhandled exception during tab completion for command '/mydog list' in plugin MyDog v0.2.0
        at org.bukkit.command.PluginCommand.tabComplete(PluginCommand.java:150) ~[craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at org.bukkit.command.Command.tabComplete(Command.java:91) ~[craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at org.bukkit.command.SimpleCommandMap.tabComplete(SimpleCommandMap.java:227) ~[craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at org.bukkit.craftbukkit.v1_15_R1.CraftServer.tabCompleteCommand(CraftServer.java:1644) ~[craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at org.bukkit.craftbukkit.v1_15_R1.CraftServer.tabComplete(CraftServer.java:1623) ~[craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at org.bukkit.craftbukkit.v1_15_R1.command.BukkitCommandWrapper.getSuggestions(BukkitCommandWrapper.java:49) ~[craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at com.mojang.brigadier.tree.ArgumentCommandNode.listSuggestions(ArgumentCommandNode.java:71) ~[craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at com.mojang.brigadier.CommandDispatcher.getCompletionSuggestions(CommandDispatcher.java:596) ~[craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at com.mojang.brigadier.CommandDispatcher.getCompletionSuggestions(CommandDispatcher.java:579) ~[craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at net.minecraft.server.v1_15_R1.PlayerConnection.a(PlayerConnection.java:517) ~[craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at net.minecraft.server.v1_15_R1.PacketPlayInTabComplete.a(SourceFile:35) ~[craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at net.minecraft.server.v1_15_R1.PacketPlayInTabComplete.a(SourceFile:9) ~[craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at net.minecraft.server.v1_15_R1.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:19) ~[craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at net.minecraft.server.v1_15_R1.TickTask.run(SourceFile:18) [craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeTask(SourceFile:144) [craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at net.minecraft.server.v1_15_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) [craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeNext(SourceFile:118) [craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at net.minecraft.server.v1_15_R1.MinecraftServer.ba(MinecraftServer.java:876) [craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at net.minecraft.server.v1_15_R1.MinecraftServer.executeNext(MinecraftServer.java:869) [craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.awaitTasks(SourceFile:127) [craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at net.minecraft.server.v1_15_R1.MinecraftServer.sleepForTick(MinecraftServer.java:853) [craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:793) [craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_371]
Caused by: java.lang.NoClassDefFoundError: org/apache/commons/lang3/Validate
        at dk.fido2603.mydog.managers.CommandManager.onTabComplete(CommandManager.java:1068) ~[?:?]
        at dk.fido2603.mydog.MyDog.onTabComplete(MyDog.java:396) ~[?:?]
        at org.bukkit.command.PluginCommand.tabComplete(PluginCommand.java:141) ~[craftbukkit-1.15.2.jar:git-Bukkit-8731c97]
        ... 22 more

--- a/pom.xml
+++ b/pom.xml
@@ -79,12 +79,12 @@
             <scope>test</scope>
         </dependency>
         <!-- apache commons lang -->
-        <dependency>
-            <groupId>org.apache.commons</groupId>
-            <artifactId>commons-lang3</artifactId>
-            <version>3.12.0</version>
-            <scope>provided</scope>
-        </dependency>
+<!--        <dependency>-->
+<!--            <groupId>org.apache.commons</groupId>-->
+<!--            <artifactId>commons-lang3</artifactId>-->
+<!--            <version>3.12.0</version>-->
+<!--            <scope>provided</scope>-->
+<!--        </dependency>-->
     </dependencies>
 
     <packaging>jar</packaging>

--- a/src/main/java/dk/fido2603/mydog/managers/CommandManager.java
+++ b/src/main/java/dk/fido2603/mydog/managers/CommandManager.java
@@ -4,7 +4,6 @@ import java.text.DecimalFormat;
 import java.util.*;
 
 import dk.fido2603.mydog.MyDog;
-import org.apache.commons.lang3.Validate;
 import org.bukkit.ChatColor;
 import org.bukkit.Location;
 import org.bukkit.attribute.Attribute;
@@ -1065,9 +1064,9 @@ public class CommandManager {
     }
 
     public List<String> onTabComplete(CommandSender sender, Command cmd, String alias, String[] args) {
-        Validate.notNull(sender, "Sender cannot be null");
-        Validate.notNull(args, "Arguments cannot be null");
-        Validate.notNull(alias, "Alias cannot be null");
+        if (sender == null || alias == null || args == null) {
+            return Collections.emptyList();
+        }
 
         List<String> result = new ArrayList<String>();
 

